Woof/Woof/Helper/Color+Extension.swift

```diff
@@ -0,0 +1,10 @@
+import SwiftUI
+
+/// Describes the color scheme using in the application
+extension Color {
+    static let whiteCustom = Color(red: 251 / 255, green: 251 / 255, blue: 251 / 255)
```

Comment:
Missing doc comments.

for example, why this white is "custom"?

Reply:
Add documentation.